http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -219,6 +222,7 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
                                     googleProject: GoogleProject,
                                     clusterName: ClusterName,
                                     clusterRequest: ClusterRequest): Future[Cluster] = {
+    val traceId = TraceId(randomUUID)
```

Comment:
I'd prefer this is being passed in so that one `traceId` can correlate as many logs as possible

Reply:
I think `processClusterCreationRequest` is the entry point for the `v2` createCluster request. `createCluster` and `internalCreateCluster` are for `v1` and deprecated. I'm actually removing them in my branch. :)

Reply:
The names are confusing indeed. Perhaps we can rename `processClusterCreationRequest` as `createCluster` once you remove the older version.

Reply:
Good idea